spk/jellyfin/src/service-setup.sh

```diff
@@ -1,9 +1,24 @@
 # https://github.com/dotnet/core/issues/4011
 [ -z "$DOTNET_BUNDLE_EXTRACT_BASE_DIR" ] && export DOTNET_BUNDLE_EXTRACT_BASE_DIR="${XDG_CACHE_HOME:-"/var/packages/jellyfin/target/var/"}/dotnet_bundle_extract"
 
-SERVICE_COMMAND="env LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${SYNOPKG_PKGDEST}/share/jellyfin --package-name synology -d ${SYNOPKG_PKGDEST}/var/data -C ${SYNOPKG_PKGDEST}/var/cache -c ${SYNOPKG_PKGDEST}/var/config -l ${SYNOPKG_PKGDEST}/var/log -w ${SYNOPKG_PKGDEST}/web"
+JELLYFIN_ARGS="--service \
+ --package-name synology \
+ -d ${SYNOPKG_PKGDEST}/var/data \
+ -C ${SYNOPKG_PKGDEST}/var/cache \
+ -c ${SYNOPKG_PKGDEST}/var/config \
+ -l ${SYNOPKG_PKGDEST}/var/log \
+ -w ${SYNOPKG_PKGDEST}/web \
+ --ffmpeg /var/packages/ffmpeg/target/bin/ffmpeg"
+
+SERVICE_COMMAND="env LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
 
 SVC_BACKGROUND=y
 SVC_WRITE_PID=y
 
 GROUP=sc-media
+
+service_postinst ()
+{
+    # allow ffmpeg access
+    chmod 775 /volume1/@appstore/jellyfin/var/data/transcodes
```

Comment:
@th0ma7 is there a better way to allow ffmpeg write access to a different package?

Reply:
Question is, what is the real issue here?  `ffmpeg` has no user.  It run as calling user.  Unless I'm not understanding something (haven't looked at the code), this can only mean that **A)** the group write access can only be for some other user part of `sc-media` calling `ffmpeg` needing to write in the `transcodes` directory?  Or **B)** is it useless as there is only `jellyfin` user that needs access to it?

Still, options could be (options 1-2 could be built into the default scripting install environment):
1. use `umask`
2. create the directory calling `mkdir --mode=0775 $dir`
3. use another `SSS_SCRIPT` which only function is to check & change directory permissions (as this is run as `root` I believe)

Reply:
The reason is jellyfin executes ffmpeg like this and causes permission errors:
```
/var/packages/ffmpeg/target/bin/ffmpeg -analyzeduration 3000000 -user_agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.85 Safari/537.36" -fflags +igndts+genpts -f hls -i "https://c.mjh.nz/101002210224/" -map_metadata -1 -map_chapters -1 -threads 0 -sn -codec:v:0 copy -start_at_zero -flags -global_header -vsync -1 -acodec copy -strict -2 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/61565f89694a65ed517103a697460d55/" -hls_playlist_type event -hls_segment_filename "/volume1/@appdata/jellyfin/data/transcodes/61565f89694a65ed517103a697460d55%d.ts" -y "/volume1/@appdata/jellyfin/data/transcodes/61565f89694a65ed517103a697460d55.m3u8"

[...]

[hls @ 0xe74d80] Opening '/volume1/@appdata/jellyfin/data/transcodes/61565f89694a65ed517103a697460d550.ts' for writing
[hls @ 0xe74d80] Failed to open file '/volume1/@appdata/jellyfin/data/transcodes/61565f89694a65ed517103a697460d550.ts'
av_interleaved_write_frame(): Permission denied

```

Reply:
Thanks I went with option 2 :+1: